arangod/ClusterEngine/ClusterIndex.cpp

```diff
@@ -421,3 +479,28 @@ ClusterIndex::coveredFields() const {
       return _fields;
   }
 }
+
+bool ClusterIndex::inProgress() const {
+  auto const& vocbase = _collection.vocbase();
+  auto const& dbname = vocbase.name();
+  auto const cid = std::to_string(_collection.id().id());
+  auto const& agencyCache =
+      vocbase.server().getFeature<ClusterFeature>().agencyCache();
+  auto [acb, idx] =
+      agencyCache.read(std::vector<std::string>{AgencyCommHelper::path(
+          "Plan/Collections/" + dbname + "/" + cid + "/indexes")});
```

Comment:
this must use URL-encoding
```suggestion
          "Plan/Collections/" + basics::StringUtils::urlEncode(dbname) + "/" + cid + "/indexes")});
```